File: src/AuthDemo.Host.Web/WebApp/modules/app/model/City.js
Ext.define('AD.model.City',{
	extend: 'Ext.data.Model',
	idProperty: 'Id',
    fields:[
    	{name: 'Id', type:'int'},
    	{name: 'Name', type:'string'},
    	{name: 'Population', type:'int'},
    	{name: 'CountryId', type:'int'}
    ]
});

File: src/AuthDemo.ServiceModel/Types/City.cs
using System;
using System.ComponentModel.DataAnnotations;
using ServiceStack.Common;
using ServiceStack.DataAnnotations;
using ServiceStack.DesignPatterns.Model;

namespace AuthDemo.ServiceModel.Types
{
	[Alias("CITY")]
	public partial class City:IHasId<System.Int32>{

		public City(){}
		
		[Alias("CityId")]
		[AutoIncrement]
		public int Id { get; set;}
		public int CountryId { get; set;}
		[StringLength(40)]
		public string Name { get; set;}
		[Alias("People")]
		public int Population{ get;set;}
		
	}
}
